microprofile/testing/testng/src/main/java/io/helidon/microprofile/testing/testng/HelidonTestNgListener.java

```diff
@@ -653,4 +704,15 @@ public Class<? extends Extension> value() {
         }
     }
 
+    /**
+     * Supports inline instantiation of the {@link Inject} annotation.
+     */
+    private static final class Literal extends AnnotationLiteral<Inject> implements Inject {
```

Comment:
This type already exists:
`jakarta.enterprise.inject.literal.InjectLiteral` there should be no need to create it here or in Junit extension, as we expect CDI to be on classpath (as otherwise this module has nothing to do)